pkg/api/validation/dynakube/logmonitoring.go

```diff
@@ -7,10 +7,23 @@ import (
 )
 
 const (
-	warningLogMonitoringIgnoredTemplate = "The Dynakube's `spec.templates.logMonitoring` section is skipped as the `spec.oneagent` section is also configured."
-	errorLogMonitoringMissingImage      = `The Dynakube's specification specifies standalone Log monitoring, but no image repository/tag is configured.`
+	warningLogMonitoringIgnoredTemplate    = "The Dynakube's `spec.templates.logMonitoring` section is skipped as the `spec.oneagent` section is also configured."
+	errorLogMonitoringMissingImage         = `The Dynakube's specification specifies standalone Log monitoring, but no image repository/tag is configured.`
+	errorLogMonitoringWithoutK8SMonitoring = "The Dynakube's specification specifies Log monitoring without an ActiveGate with kubernetes-monitoring enabled or the automatic-kubernetes-monitoring feature flag. This is not allowed as Kubernetes settings are needed for Log monitoring."
 )
 
+func logMonitoringWithoutK8SMonitoring(ctx context.Context, dv *Validator, dk *dynakube.DynaKube) string {
+	if dk.LogMonitoring().IsEnabled() && !dk.ActiveGate().IsKubernetesMonitoringEnabled() {
+		if !dk.FeatureAutomaticKubernetesApiMonitoring() {
+			return errorLogMonitoringWithoutK8SMonitoring
+		}
+
+		return ""
```

Comment:
Don't think this is right. 

What is assumed here is that `FeatureAutomaticKubernetesApiMonitoring` will always create the kubernetes settings if turned on. In the current implementation this only happens if kubemon capability is turned on.

If kubernetes monitoring is disabled, automatic kubemon ff does nothing - The setting will never be created.

Reply:
I assumed that `FeatureAutomaticKubernetesApiMonitoring` ensures that there is a Kubernetes setting. Isn't it what it is doing?

If this is not the case we may think about changing the description

Reply:
Only if the ActiveGate kubemon cap is turned on, look [here](https://github.com/Dynatrace/dynatrace-operator/blob/6900fe53167898c1a32e41e8bb650a5423aa6747/pkg/controllers/dynakube/activegate.go#L28).

Problem is that a MEID is required before the setting can be created I think. And the MEID is only created in case the kubemon ActiveGate is running. - Not 100% sure about that though 🤔

Reply:
Yes I think you are right, AFAIK the MEID is required to create settings. Then I guess I interpreted the description of the FF wrong. 

Now that I think about it, it can be left out completely in the validation, right?

Reply:
I think we have to look at the following 4 cases: 
- Kubemon enabled, ff enabled: Settings object will be there, logmonitoring validation should pass
- Kubemon enabled, ff disabled: Settings object will be created by customer (hopefully), logmonitoring validation should pass
- kubemon disabled, ff enabeld: Settings object will not be created, logmonitoring validation should fail
- kubemon disabled, ff disabled: Settings object will be created by customer, logmonitoring should pass

If the ff can be interpreted from customer perspective as: "Yes I created settings, let me do stuff on my own"

Reply:
I am not sure about `kubemon disabled, ff disabled` as this would contradict the decision that was made to couple `logMonitoring` and `kubernetes-monitoring` capability together and make the requirement more obvious. 

On the other hand maybe a nicely formulated validation warning might do the trick here. I am unsure and would suggest to wait for @0sewa0 's feedback here, as he had discussions with PM's that led to the decision to make the requirement `kubernetes-monitoring `for `logMonitoring` more strict.

Reply:
done https://github.com/Dynatrace/dynatrace-operator/pull/4309/commits/e77ae8f5161cf5ae44e6edb5ab3d2bb3c37e01bf